frontend/public/components/events.jsx

```diff
@@ -474,32 +506,36 @@ class EventStream extends React.Component {
       sysEventStatus = <NoEvents />;
     }
     if (noMatches && !resourceEventStream) {
-      sysEventStatus = <NoMatchingEvents />;
+      sysEventStatus = <NoMatchingEvents allCount={allCount} />;
     }
 
     if (error) {
       statusBtnTxt = (
         <span className="co-sysevent-stream__connection-error">
-          Error connecting to event stream{_.isString(error) && `: ${error}`}
+          {t('events~Error connecting to event stream')}
+          {_.isString(error) && t('events~: {{error}}', { error })}
```

Comment:
This shouldn't be localized. Just show `error` as-is.

Reply:
@spadgett the error isn't being localized but i'm going to make this more clear.